Packages/com.unity.inputsystem/InputSystem/Plugins/InputForUI/InputActionAssetVerifier.cs

```diff
@@ -32,9 +32,23 @@ static InputActionAssetVerifier()
         public void Verify(InputActionAsset asset,
             ProjectWideActionsAsset.IReportInputActionAssetVerificationErrors reporter)
         {
-            // Note that we never cache this to guarantee we have the current configuration.
-            var config = InputSystemProvider.Configuration.GetDefaultConfiguration();
-            Verify(asset, ref config, reporter);
+            // Note:
+            // PWA has initial state check true for "Point" action, DefaultActions do not, does it matter?
```

Comment:
Not sure, but I believe it makes more sense to have it on than off

Reply:
This comment was copied from a previous change.